deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,75 @@
+{
+  'variables': {
+    'arm_fpu%': '',
+    'target_arch%': '',
+  },
+
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'direct_dependent_settings': {
+        'include_dirs': ['.'],
+      },
+      'sources': ['simdutf.cpp'],
+      'conditions': [
+        [ 'arm_fpu=="neon" and target_arch=="arm"', {
+          'conditions': [
+            [ 'OS!="win"', {
+              'cflags': [ '-mfpu=neon' ],
+              'xcode_settings': {
+                'OTHER_CFLAGS': [ '-mfpu=neon' ]
+              },
+            }],
+          ],
+        }],
+
+        # Runtime detection will happen for x86 CPUs
+        [ 'target_arch in "ia32 x64 x32"', {
+          'conditions': [
+            [ 'OS!="win"', {
+              'cflags': [
+                '-msse4.2',
+                '-mavx',
+                '-mavx2',
+                '-mavx512f',
+                '-mavx512dq',
+                '-mavx512pf',
+                '-mavx512er',
+                '-mavx512cd',
+                '-mavx512bw',
+                '-mavx512vl',
+                '-mavx512vbmi2',
```

Comment:
@daeyeon Are these AVX512 flags correct?

Reply:
simdutf requires AVX512-VBMI2, and I found [a `.gyp` example](https://github.com/animetosho/node-yencode/blob/748164de23f5f2e28b581329f52f14ebb25fa85b/binding.gyp#L256) which detects VBMI2, but couldn't port it to `simdutf` example. Appreciate any help on this.